modin/experimental/xgboost/xgboost.py

```diff
@@ -94,23 +100,18 @@ def __init__(self, params=None, cache=(), model_file=None):  # noqa: MD01
     def predict(
         self,
         data: DMatrix,
-        num_actors: Optional[int] = None,
         **kwargs,
     ):
         """
         Run distributed prediction with a trained booster.
 
-        During work it evenly distributes `data` between workers,
-        runs xgb.predict on each worker for subset of `data` and creates
-        Modin DataFrame with prediction results.
+        During work it runs xgb.predict on each worker for row partition of `data`
```

Comment:
```suggestion
        During execution it runs `xgb.predict` on each worker for subset of `data`
```

Reply:
Used the next
```python
During execution it runs ``xgb.predict`` on each worker for subset of `data`
```
because we use one backtick for parameters highlighting, double for code entities